0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -191,106 +150,21 @@ Content-Type: application/json
 }
 ```
 
-##### Invoice
+### Payment Setup
 
-Invoice information includes an exact amount as well as the status of the invoice. (Invoices can only be paid once.)
+The sender uses the receiver details to create the ILP packet:
+* The `account` in the ILP packet is the `destination_account` provided by the receiver
+* The `amount` is determined by the sender:
 
-Example Receiver:
-``` json
-{
-  "type": "invoice",
-  "account": "ilpdemo.red.amazon.111-7777777-1111111",
-  "currency_code": "USD",
-  "currency_symbol": "$",
-  "amount": "10.40",
-  "status": "unpaid",
-  "invoice_info": "https://www.amazon.com/gp/your-account/order-details?ie=UTF8&orderID=111-7777777-1111111"
-}
-```
+  * The sender MAY use the `ledger_info.scale` to determine the integer amount to include in the ILP packet
+  * The sender SHOULD NOT set an amount greater than the `maximum_destination_amount` or the `ledger_info.precision` or one lower than the `minimum_destination_amount`, as this will be rejected by the receiver
```

Comment:
> an amount greater than the `maximum_destination_amount` or the `ledger_info.precision`

Sounds like `ledger_info.precision` is a max amount instead of a number of digits. I would explicitly call out the difference, e.g. "an amount greater than the `maximum_destination_amount` or with a precision greater than `ledger_info.precision`."

Maybe split the precision thing into another sentence.